examples/src/main/scala/org/apache/spark/examples/ml/AFTSurvivalRegressionExample.scala

```diff
@@ -59,4 +59,4 @@ object AFTSurvivalRegressionExample {
     sc.stop()
   }
 }
-// scalastyle:off println
+// scalastyle:on println
```

Comment:
These won't matter, since it's the end of the file, but I like the correctness/consistency.